scripts/linters/custom_eslint_checks/index.js

```diff
@@ -0,0 +1,24 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Indexes all custom eslint rules.
+ */
+
+'use strict';
+
+var requireIndex = require('requireindex');
+
+// Import all rules in lib/rules.
```

Comment:
Update this comment as per new dir structure*

Reply:
Done.